apps/air-discount-scheme/web/components/Header/Header.tsx

```diff
@@ -39,16 +36,13 @@ function Header({ routeKey, localeKey }: PropTypes) {
   const { toRoute, activeLocale, switchLanguage } = useI18n()
 
   const nextLanguage = activeLocale === 'is' ? 'en' : 'is'
+  const nextPath = activeLocale === 'is' ? '/' : '/en'
```

Comment:
This should actually be `'is' ? 'https://island.is/loftbru' : 'https://island.is/en/lower-airfares-for-residents-in-rural-areas'`

Reply:
The redirect will handle that for us

Reply:
aaaahhh sooooo smaaaaart

Reply:
What about local development?